modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx

```diff
@@ -354,10 +379,38 @@ export function ModalAddFilter({
 		}
 	}, [editingFilter, editingObjectFieldName, objectFields]);
 
+	const isMultiSelectValue = () => {
+		if (
+			aggregationFilter &&
+			selectedFilterBy?.businessType === 'Relationship'
+		) {
+			return false;
+		}
+
+		if (
+			selectedFilterTypeValue &&
+			(selectedFilterBy?.name === 'status' ||
+				selectedFilterBy?.businessType === 'MultiselectPicklist' ||
+				selectedFilterBy?.businessType === 'Picklist' ||
+				selectedFilterBy?.businessType === 'Relationship')
+		) {
+			return true;
+		}
+	};
+
```

Comment:
Hey @igor-franca, I understand! In this case, we organized it this way because of error the **sf** was pointing out: `'isMultiSelectValue' was used before it was defined.`

Reply:
Thanks for the explanation 😆